lib/autokey/gtkapp.py

```diff
@@ -291,6 +291,8 @@ def show_script_error(self, parent):
         if self.service.scriptRunner.error != '':
             dlg = Gtk.MessageDialog(type=Gtk.MessageType.INFO, buttons=Gtk.ButtonsType.OK,
                                      message_format=self.service.scriptRunner.error)
+            dlg.get_children()[0].get_children()[0].get_children()[1].get_children()[0].set_selectable(True)
```

Comment:
What does this do? The change to the xml file was all that it needed for me.

Reply:
It allows selecting the error in the message dialog.